pyfar/classes/orientations.py

```diff
@@ -3,11 +3,15 @@
 import warnings
 
 import pyfar as pf
+if np.__version__ < '2.0.0':
+    from numpy import VisibleDeprecationWarning
+else:
+    from numpy.exceptions import VisibleDeprecationWarning
```

Comment:
This class has been moved to the exceptions module in numpy 1.25 (https://numpy.org/doc/stable/release/1.25.0-notes.html#numpy-now-has-an-np-exceptions-namespace), currently we're supporting versions >= 1.23.
So another option to avoid this would be raise the minimum supported numpy version to 1.25.
Downside is that numpy 1.25 also requires at minimum Python 3.9.